### ISSUE TITLE
AttributeError: module 'argparse' has no attribute 'BooleanOptionalAction'

### ISSUE THREAD

**Issue by Anonymous-computer** [hippotese/datafficheur#7]
Au sujet des problèmes que j'ai rencontrés _(python 3.9 sur Linux Ubuntu 20.04.6 LTS)_.

```
--- Erreur dans la console : ---
Traceback (most recent call last):
File "datafficheur-2023-06-08.py", line 48, in 
action=argparse.BooleanOptionalAction,
AttributeError: module 'argparse' has no attribute 'BooleanOptionalAction'
---Fin erreur ---
```

Sur la proposition de @fxprunayre, j'ai mis en commentaire la ligne 43
`#action=argparse.BooleanOptionalAction,`
Et plus d'erreur...

**Comment by Anonymous-computer** [comment]
Bonjour.

Cela ressemble à un attribut non implémenté, pourtant la [doc](https://docs.python.org/3/library/argparse.html#action) stipule que `BooleanOptionalAction` a était ajouté à partir de la version 3.9.

Le fait de commenter la ligne résous le problème, mais inhibe par là même le contrôle de la valeur qui est passé en argument. Rien de grave si la valeur est correcte _(ici `False` ou `True`)_

**Comment by fxprunayre** [comment]
Nous pouvons peut être juste ajouter une ligne dans le README indiquant qu'il est nécessaire d'avoir python >=3.9 ?

**Comment by Anonymous-computer** [comment]
Oui, c'est une bonne idée d'indiquer la version minimale dans le README.

Par contre, ce qui me semble étrange, c'est que [Hippotese-Deny-Fady](https://github.com/Hippotese-Deny-Fady) semble utiliser une version _(3.9)_ qui devrait implémenter `BooleanOptionalAction` !?
